config/jobs-chromeos.yaml

```diff
@@ -56,6 +57,7 @@ _anchors:
     rules: &kbuild-gcc-12-arm64-chromeos-rules
       tree:
       - '!android'
+      - '!chromiumos'
```

Comment:
Naming scheme for the set of key anchors in this config file might need an update in the future: now there's a ChromeOS job that is set to avoid ChromiumOS kernel tree - something to address later on.

Reply:
I agree the current naming is a bit confusing (maybe `chromeoscfg` would be a better infix on kbuild jobs for non-ChromeOS kernels with the ChromeOS configs) . I can address this in a separate PR.